### PR TITLE
RDoc-1491-2 HiLo-ID-Generation - Edited for clarity - In 5.2 I added …

### PR DESCRIPTION
**Pull request by serezhub**
…a section about manual ID generation

Due to a git push error I got, I re-cloned and copy/pasted the project into a clean project.
To see github comments until now, see the old PR at https://github.com/ravendb/docs/pull/1541

### CODE REVIEW

File: Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+{NOTE: }
+
+* The HiLo algorithm is an efficient solution used by [a session](../session/what-is-a-session-and-how-does-it-work) 
+  to generate the numeric parts of unique document identifiers. 
+  It is responsible for providing numeric values that are combined with collection names and node tags 
+  to create identifiers like `orders/10-A` or `products/93-B`. 
+
+* The HiLo algorithm is the default means of creating unique IDs in RavenDB. It is created 
+  by not specifying an `Id` value when creating a new document,
```
Comment (Danielle9897) [inline Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown:13]: It is created by not specifying an `Id` value when creating a new document
=>
These IDs are generated when creating a new document without specifying an 'Id' value.

File: Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+{CODE return_hilo_2@ClientApi\DocumentIdentifiers\HiloAlgorithm.cs /}
+
+{PANEL/}
+
+{INFO: Identity Parts Separator}
+By default, document IDs created by the server use the character `/` to separate their components.  
+This separator can be changed to any other character except `|` in the 
+[Document Store Conventions](../../client-api/configuration/conventions#changing-the-identity-separator).  
+{INFO/}
```
Comment (Danielle9897) [inline Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown:102]: This info is important - but it is hidden...
Maybe add a link to it in the 'in this page'

File: Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+the ID is unique across the cluster.  
+
+However, you can use the HiLo algorithm and create documents with your own, manually created ID that is based on the next HiLo ID number
+provided by the client.
+
+The manual generator does not guarantee unique IDs across a cluster because it does not attach a collection name and a node-tag to the ID.  
+It only provides the next number in the range that it got from the server.
+
+{WARNING: }
+[RavenDB can only guarantee unique IDs when using RavenDB's ID generators.](../../client-api/document-identifiers/working-with-document-identifiers)
```
Comment (Danielle9897) [inline Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown:117]: Don't make the whole sentence 'red'...
Place the link only on 'RavenDB's ID generators'

File: Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+| Return Value | Type | Description |
+| --- | -- | --------|
+| **nextId** | `long` | The next available number from the HiLo range reserved for the client. |
+
+#### Examples: Manual HiLo Generators
+
+The following example shows how to get the next HiLo ID number from the client.  
+This ID is then used to create and store a new document.  
+The ID provided is the next unique number without the node tag and the collection.
```
Comment (Danielle9897) [inline Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown:144]: Reverse sentences order:

first:
The ID provided is the next...

second:
This ID is then used to create...

**Review by Danielle9897 — CHANGES_REQUESTED** [review]

File: Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown (hunk 1 above)
Comment (serezhub) [reply Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown:13]: done

File: Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown (hunk 2 above)
Comment (serezhub) [reply Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown:102]: done.

File: Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown (hunk 3 above)
Comment (serezhub) [reply Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown:117]: done

File: Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown (hunk 4 above)
Comment (serezhub) [reply Documentation/5.2/Raven.Documentation.Pages/client-api/document-identifiers/hilo-algorithm.dotnet.markdown:144]: done

**Review by Danielle9897 — APPROVED** [review]

**Pull request merged**